base/sparse/sparsematrix.jl

```diff
@@ -1553,6 +1553,19 @@ end
 
 sparse(S::UniformScaling, m::Integer, n::Integer=m) = speye_scaled(S.λ, m, n)
 
+Base.iszero(A::SparseMatrixCSC) = iszero(view(A.nzval, 1:(A.colptr[size(A, 2) + 1] - 1)))
```

Comment:
Would it be worthwhile to add a test for `iszero` that would fail without the `view(A.nzval, 1:(A.colptr[size(A, 2) + 1] - 1))` part? (i.e. where simply `iszero(A.nzval)` is `false`)

Reply:
Absolutely, and done! Thanks! :)